Cargo.toml

```diff
@@ -1,7 +1,20 @@
 [package]
 name = "license-exprs"
-version = "1.3.0"
+version = "1.4.0"
```

Comment:
We'll want to bump the version before we push a new release (hopefully soon), but I'd rather have version bumps in their own pull requests, not mixed in with logic changes.  Can you leave this at 1.3.0 for this PR?